tools/cpp/windows_cc_configure.bzl

```diff
@@ -93,6 +93,36 @@ def find_vc_path(repository_ctx):
     _auto_configure_warning_maybe(repository_ctx, "'BAZEL_VC' is not set, " +
                                                   "start looking for the latest Visual C++ installed.")
 
+    if "BAZEL_VC_TOOL" in repository_ctx.os.environ:
```

Comment:
Please run [Buildifier](https://github.com/bazelbuild/buildtools/releases) on the changed `.bzl` files.